actions/juju-k8s-crashdump/action.yaml

```diff
@@ -30,6 +30,9 @@ runs:
     with:
       repository: canonical/juju-k8s-crashdump
       path: ./juju-k8s-crashdump
+      ref: ${{ env.ACTION_REF }}
+    env:
+      ACTION_REF: ${{ github.action_ref }}
```

Comment:
Round about way of passing this in is necessary, see https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#github-context